packages/gdl-frontend/components/Level/LevelRibbon.js

```diff
@@ -22,6 +22,7 @@ const colorMap = {
   '3': '#F1C528',
   '4': '#FA9F28',
   'read-aloud': '#F56324',
+  'new-arrivals': '#D2D2D2',
```

Comment:
Type `ReadingLevel` does not include `new-arrivals`, so this color wil actually never be used.